cmd/release-controller/http.go

```diff
@@ -464,6 +465,42 @@ func (c *Controller) httpReleaseChangelog(w http.ResponseWriter, req *http.Reque
 	fmt.Fprintln(w, out)
 }
 
+func (c *Controller) httpReleaseInfoJson(w http.ResponseWriter, req *http.Request) {
+	start := time.Now()
+	defer func() { glog.V(4).Infof("rendered in %s", time.Now().Sub(start)) }()
+
+	tag := req.URL.Query().Get("tag")
+	if len(tag) == 0 {
+		http.Error(w, fmt.Sprintf("tag must be specified"), http.StatusBadRequest)
+		return
+	}
+
+	tags, ok := c.findReleaseStreamTags(false, tag)
+	if !ok {
+		for k, v := range tags {
+			if v == nil {
+				http.Error(w, fmt.Sprintf("could not find tag: %s", k), http.StatusBadRequest)
+				return
+			}
+		}
+	}
+
+	tagBase := tags[tag].Release.Target.Status.PublicDockerImageRepository
+	if len(tagBase) == 0 {
+		http.Error(w, fmt.Sprintf("release target %s does not have a configured registry", tags[tag].Release.Target.Name), http.StatusBadRequest)
+		return
+	}
+
+	out, err := c.releaseInfo.ReleaseInfo(tagBase+":"+tag)
```

Comment:
To look up the image, you have to find it in the target status tags. You should not have to construct a pull spec directly.